pulsar-common/src/main/java/org/apache/pulsar/common/util/collections/ConcurrentLongHashMap.java

```diff
@@ -333,25 +334,23 @@ V get(long key, int keyHash) {
                         if (!acquiredLock) {
                             stamp = readLock();
                             acquiredLock = true;
+
+                            // update local variable
+                            keys = this.keys;
+                            values = this.values;
+                            bucket = signSafeMod(keyHash, capacity);
```

Comment:
shouldn't `values.length` be used instead of `this.capacity`?
```suggestion
                            bucket = signSafeMod(keyHash, values.length);
```

Reply:
fixed.